src/utils/utils.go

```diff
@@ -11,8 +11,8 @@ import (
 
 	"github.com/openshift/assisted-installer-agent/pkg/journalLogger"
 
-	ignition "github.com/coreos/ignition/config/v2_2"
-	"github.com/coreos/ignition/config/v2_2/types"
+	ignition "github.com/coreos/ignition/v2/config/v3_1"
```

Comment:
This break current (4.5) installation right?

Reply:
right, but we won't support it once we move to 4.6